contracts/deploy/L2Endpoint/L2Endpoint.cjs

```diff
@@ -1,15 +1,28 @@
+const path = require('node:path')
+const { loadJson, loadMigration, updateMigration } = require('../../scripts/v0.1/utils.cjs')
+
 const func = async function (hre) {
   const { deployments, getNamedAccounts, network } = hre
   const { deploy } = deployments
   const { deployer } = await getNamedAccounts()
+  const migrationDirPath = `./migration/${network.name}/L2Endpoint`
```

Comment:
I think this does not correspond with the second file in this PR which is located at `migration/l2node/L2Endpoint` path which means it would not work. Can you please change the path from `contracts/migration/l2node/L2Endpoint/20231107150650_deploy.json` to `contracts/migration/baobab/L2Endpoint/20231107150650_deploy.json` since you connected it with baobab?

Reply:
This L2endpoint is deployed on the L2 node. we don't deploy it on baobab, or am I missing your point?

Reply:
The `migrationDirPath` defines where to look for migration files. Your migration file has different path.

Reply:
Please ignore this comment 🙏